README.md

```diff
@@ -157,7 +157,7 @@ So, you can type in any username to get access to the desktop, which likely does
 Read the [Configuration](https://github.com/zowe/zlux/wiki/Configuration-for-zLUX-App-Server-&-ZSS) wiki page for a detailed explanation of the primary items that you'll want to configure for your server.
 
 In short, determine the location of your workspace directory (Environment variable `ZWE_zowe_workspaceDirectory`, or `~/.zowe/workspace` (Linux, Unix, z/OS) or `%USERPROFILE%/.zowe` when `ZWE_zowe_workspaceDirectory` is not defined)
-Within the workspace directory, edit **app-server/serverConfig/server.json** to change attributes such as the HTTPS port via **node.https.port**, or the location of plugins.
+Within the workspace directory, edit **app-server/serverConfig/zowe.yaml** to change attributes such as the HTTPS port via **components.app-server.node.https.port**, or the location of plugins.
```

Comment:
My workspace directory (home-dir/zowe-2.0.0-pr-2578/instance/workspace/app-server/serverConfig) does not have a zowe.yaml

Are you sure this is accurate?

Reply:
Just saw the comment in https://github.com/zowe/zlux-app-server/pull/199/files#diff-29da904bd638a6675cfd1196b4d4e3c52fd059d93f49d457d85b9c56a1c501a8R52
seems to prove my point